### PR TITLE
Clojure scaffold

### PR DESCRIPTION
**Pull request by jbruggem**

### CODE REVIEW

File: scaffolds/clojure/README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Javascript Scaffold for Code Retreat
```
Comment (jbruggem) [inline scaffolds/clojure/README.md]: JS => Clojure

File: scaffolds/clojure/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+## Install Leiningen
+
+[Follow the instructions](http://leiningen.org/#install).
```
Comment (jbruggem) [inline scaffolds/clojure/README.md]: + also mention need for JVM

File: scaffolds/clojure/test/scaffold/core_test.clj [hunk 3]
```diff
@@ -0,0 +1,7 @@
+(ns scaffold.core-test
+  (:require [clojure.test :refer :all]
+            [scaffold.core :refer :all]))
+
+(deftest a-test
```
Comment (toch) [inline scaffolds/clojure/test/scaffold/core_test.clj]: Do you still need that one?

**Comment by toch** [comment]
Thanks @jbruggem.

Could you keep the minimum into `.gitignore`?

File: scaffolds/clojure/test/scaffold/core_test.clj (hunk 3 above)
Comment (jbruggem) [inline scaffolds/clojure/test/scaffold/core_test.clj]: What are you referring to ?

Comment (toch) [inline scaffolds/clojure/test/scaffold/core_test.clj]: it's seems to not test foo, am I wrong?

**Comment by jbruggem** [comment]
Ok, everything should be addressed. You're welcome to merge if you're satisfied :)

**Comment by toch** [comment]
Awesome :clap: 
Time to :shipit: then!

**Pull request merged**